From 4e2895b8118ef5eb160179f5b07b196fb137b608 Mon Sep 17 00:00:00 2001
From: Holger Hans Peter Freyther <zecke@selfish.org>
Date: Wed, 18 Mar 2009 17:36:14 +0800
Subject: [PATCH] Hardcode language to english now...

Instead of looking at the HTTP header hardcode the
language to english...
---
 merb/extensions/request_extensions.rb |    5 +----
 1 files changed, 1 insertions(+), 4 deletions(-)

diff --git a/merb/extensions/request_extensions.rb b/merb/extensions/request_extensions.rb
index 830318b..f56a5bb 100644
--- a/merb/extensions/request_extensions.rb
+++ b/merb/extensions/request_extensions.rb
@@ -13,10 +13,7 @@ class Merb::Request
   
   # Gets the language code for this request
   def language_code
-    language_code = host.split(".").first
-    if Merb.env?(:test) || language_code.include?("localhost") || language_code == "eiximenis"
-      language_code = "en"
-    end
+    language_code = "en"
     params[:lang]= language_code
     language_code
   end
-- 
1.5.6.3

